### ISSUE TITLE
URL filter is not supported

### ISSUE THREAD

**Issue by giedriuswix** [wix/velo-external-db#222]
This prevents using External DB Drivers in Dynamic Pages with URL binding.

Tested in MySQL.

**Comment by Idokah** [comment]
@giedriuswix, do you mean the `urlized` operator?

**Comment by Idokah** [comment]
Update on this issue: 
*urlized* is not an operator, which we did not know until now. It is just an announcement that the field is supporting these two operators:

1. `in` operator - we are already supporting this.
2. `matches` operator - we weren't aware that this operator existed, and there is no documentation for it. However, from what I understood from @giedriuswix, it works this way: 

example : 
```
{
field: {
  $matches: {
    ignoreCase: true, // whether match should be case-insensitive
    spec: [
      { type: "literal", value: "test" },
      { type: "anyOf", value: " \t\n-" },
      { type: "literal", value: "more" }
    ]
  }
}
```
It should create a query that finds values in the field that match the following regex : `/test[ \t\n-]more/` 

`spec` is an array of objects of the following type : 

- 'literal' - match the exact value
- 'anyOf' - match as regex the value.

I've added support for this operator to MySQL #233.
I tried to check this with a wix-data query, but I did not succeed. Giedrius estimates it is a wix-data issue.
As soon as the bug is fixed, I'll check the operator in MySQL, and implement it for all other implementations.
